### ISSUE TITLE
AuthorRetrieval.historical_identifier is empty although AuthorRetrieval shows alternative Scopus IDs in stdout

### ISSUE THREAD

**Issue by raffaem** [pybliometrics-dev/pybliometrics#163]
I downloaded the publications of author ID '7005779708'. He is called Sjöberg, Jonas.

This is the gist of the code I used to do so:

```
    query = f"AU-ID({author_id})"
    res = ScopusSearch(query, verbose=True, refresh=True)  
```

Unfortunately in the publication data the author always appears with SCOPUS ID '57218613709'. This prevents me from identifying the name and the affiliation as it appears in the paper.

This "alternative SCOPUS ID" **does appear** in stdout when I call AuthorRetrieval on the "original" SCOPUS ID. 

I thought that this "alternative SCOPUS ID" would also appear in `AuthorRetrieval.historical_identifier`, but it does not.

Here is an example:

```
>>> from pybliometrics.scopus import AuthorRetrieval
>>> au = AuthorRetrieval(7005779708)
C:\Users\Raffaele\anaconda3\lib\site-packages\pybliometrics\scopus\author_retrieval.py:236: UserWarning: Author profile with ID 7005779708 has been merged and the main profile is now one of 57193605516, 57218613713, 57218613712, 36778463000, 57212986457, 57218613709.  Please update your records manually.  Functionality of this object is reduced.
  warn(text, UserWarning)
>>> au.identifier
'7005779708'
>>> au.historical_identifier
>>>
``

The  `.historical_identifier` property is empty.

How I can obtain from the code all the alternative SCOPUS IDs for this author? Said otherwise, how I can obtain from the code what pybliometrics prints in stdout when I call AuthorRetrieval?

**Comment by Michael-E-Rose** [comment]
The user warning in fact states it all: "Functionality of this object is reduced." In the Author Retrieval API, Scopus sends information about the _new_ profile, but not all, and not all the time the same information (for forwarded profiles).

What I typically do to find out the new ID is to use scopus.com. I enter the old ID directly into the URL, e.g. https://www.scopus.com/authid/detail.uri?authorId=7005779708. It forwards immediately to https://www.scopus.com/authid/detail.uri?authorId=36778463000. 

In this particular case however the profile with ID [57218613709](https://www.scopus.com/authid/detail.uri?authorId=57218613709) still exists. Therefore you see publications with that ID.

PS: You can actually ask such kind of questions as well on Stack Overflow.There's a [pybliometrics](https://stackoverflow.com/questions/tagged/pybliometrics) tag that I'm watching.

**Comment by raffaem** [comment]
Hello,

sorry If I wasn't extremely clear.

I have a large database of publications downloaded using pybliometrics. I have to parse all the publications to retrieve for each author, its affiliation for that publication.

To do so I rely on the SCOPUS ID I originally have, and which I used to download the publications.

Unfortunately the SCOPUS ID of the author _inside_ the publication is different from the one I used to download the publication.

So I am not able to identify the author inside the publication.

Now I need to retrieve the "alternative IDs" from the code. 

As you already said, moreover, even by inserting the "old ID" in scopus, 7005779708, you are redirected to yet another SCOPUS ID, 36778463000, whiel the SCOPUS ID in the publication is 57218613709.

Now AuthorRetrieval prints in stdout a list of alternative SCOPUS IDs, among which there is the correct one. If I was this list from the code, I can check whether any of the SCOPUS ID contained in there is in the publication I am analyzing, and thus find my author.

In `author_retrieval.py`, the relevant code seems to be at line 229-236:

```
            alias_json = listify(self._json['alias']['prism:url'])
            alias = ', '.join([d['$'].split(':')[-1] for d in alias_json])
            text = f'Author profile with ID {author_id} has been merged and '\
                   f'the main profile is now one of {alias}.  Please update '\
                   'your records manually.  Functionality of this object is '\
                   'reduced.'
            warn(text, UserWarning)
```

Would it be possible to save `alias` as a property of AuthorRetrieval, so that I can retrieve it in my code?
